html/includes/api_functions.inc.php

```diff
@@ -526,6 +526,29 @@ function get_bgp()
 }
 
 
+function list_cbgp()
+{
+    $app        = \Slim\Slim::getInstance();
+    $sql        = '';
+    $sql_params = array();
+    $hostname   = $_GET['hostname'] ?: '';
+    $device_id  = ctype_digit($hostname) ? $hostname : getidbyname($hostname);
+    if (is_numeric($device_id)) {
```

Comment:
We should error if this doesn't return true.

Reply:
Will fix in followup